docs/reference/migration/migrate_7_8.asciidoc

```diff
@@ -5,49 +5,65 @@
 ++++
 
 This section discusses the changes that you need to be aware of when migrating
-your application to Elasticsearch 7.8.
+your application to {es} 7.8.
 
 See also <<release-highlights>> and <<es-release-notes>>.
 
 coming[7.8.0]
 
+* <<breaking_78_mappings_changes>>
+* <<breaking_78_settings_changes>>
+
 //NOTE: The notable-breaking-changes tagged regions are re-used in the
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
 
-//end::notable-breaking-changes[]
-
-[discrete]
-[[breaking_78_api_changes]]
-=== API changes
-
-[discrete]
-==== Deprecation of `local` parameter for get field mapping API
-
-The `local` parameter is a no-op and field mappings are always retrieved locally.
-As of 7.8, this parameter is deprecated and will be removed in next major version.
-
 [discrete]
 [[breaking_78_mappings_changes]]
 === Mappings changes
 
-[discrete]
 [[prevent-enabled-setting-change]]
-==== `enabled` setting cannot be changed on root mapper
+.The `enabled` mapping parameter cannot be changed for a root mapping.
+[%collapsible]
+====
+*Details* +
+Mapping requests that attempt to change the {ref}/enabled.html[`enabled`]
+mapping parameter for a root mapping will fail and return an error.
+
+Previously, {es} accepted mapping requests that attempted to change the
+`enabled` parameter of the root mapping. Theses changes were not applied, but
+such requests didn't return an error.
+
+*How to migrate* +
```

Comment:
I prefer Kibana's more generic "Impact".  IMO you're not telling someone "how to migrate/upgrade" in this section, you're telling them "what do I have to do about it". 

```suggestion
*Impact* +
```

Reply:
Changed throughout with 20db1c3. Thanks!